packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -27,7 +26,7 @@ export type TorrentCallback = (torrent: Torrent) => any;
 export * from './types';
 
 export const WEI_PER_BYTE = bigNumberify(1); // cost per credit / piece
-export const BUFFER_REFILL_RATE = bigNumberify(100000); // number of credits / pieces the leecher wishes to increase the buffer by
+export const BUFFER_REFILL_RATE = bigNumberify(80000); // number of credits / pieces the leecher wishes to increase the buffer by
```

Comment:
Perhaps we should put this inside a `constants.ts` file so they're more easily understood to be app-level constants.